File: Odd-Even Sort/OddEvenSort.cs
internal static int[] OddEvenSort(int[] arr)
{
    for(int i = 0; i < arr.Length; ++i)
        for (int j = ((i % 2 == 0) ? 0 : 1); j + 1 < arr.Length; j += 2)
            if (arr[j] > arr[j + 1])
            {
                int temp = arr[j];
                arr[j] = arr[j + 1];
                arr[j + 1] = temp;
            }
    return arr;
}

File: Selection Sort/OddEvenSort.py
def OddEvenSort(arr):
    for i in range(len(arr)):
        for j in range(0 if i % 2 == 0 else 1, len(arr), 2):
            if j + 1 < len(arr) and arr[j] > arr[j+1]:
                arr[j], arr[j+1] = arr[j+1], arr[j]
    return arr